app/views/root/local_transaction.html.erb

```diff
@@ -58,3 +58,17 @@
     </div>
   </section>
 <% end %>
+
+<% if @postcode_error %>
+  <script type="text/javascript">
+    //<![CDATA[
+      GOVUK.analytics.trackEvent(
+        "userAlerts:LocalTransaction",
+        "postcodeErrorShown:<%= @postcode_error %>",
```

Comment:
These strings came from the Analyst on our team. I'll send him a link to this comment and see what he says.